src/components/tooltip/tooltip.e2e.ts

```diff
@@ -11,6 +11,8 @@ describe("calcite-tooltip", () => {
     });
   });
 
+  it("honors hidden attribute", async () => hidden("calcite-tooltip"));
```

Comment:
Remove. This is a duplicate test (missing `import` in the diff was the giveaway).